src/test/java/uk/gov/companieshouse/scanupondemand/orders/api/controller/ScanUponDemandItemControllerIntegrationTest.java

```diff
@@ -270,6 +264,35 @@ void getScanUponDemandItemReturnsNotFound() throws Exception {
                 .andDo(MockMvcResultHandlers.print());
     }
 
+    @Test
+    @DisplayName("Fails to create scan upon demand item that fails validation")
+    void createScanUponDemandItemFailsToScanUponDemandItem() throws Exception {
```

Comment:
Test method name `createScanUponDemandItemFailsToScanUponDemandItem` is missing second `...Create...`?